artichoke-backend/src/convert/array.rs

```diff
@@ -1,4 +1,5 @@
-// use std::collections::HashMap;
+#[cfg(feature = "artichoke-all-converters")]
+use std::collections::HashMap;
```

Comment:
if this `use` statement is only required by the optional converters which are nested in a module, can you add that use directive into that module?

As it stands, I don't know if this compiles.